server/chat-plugins/helptickets.ts

```diff
@@ -17,10 +17,18 @@ interface TicketState {
 	created: number;
 	claimed: string | null;
 	ip: string;
+	resolution?: 'unknown' | 'dead' | 'unresolved' | 'resolved';
+	result: TicketResult | null;
+	claimQueue?: string[];
+	involvedStaff?: ID[];
+	createTime?: number;
+	emptyRoom?: boolean;
+	firstClaimTime?: number;
+	unclaimedTime?: number;
 }
 type TicketResult = 'approved' | 'valid' | 'assisted' | 'denied' | 'invalid' | 'unassisted' | 'ticketban' | 'deleted';
 
-const tickets: {[k: string]: TicketState} = {};
+export const tickets: {[k: string]: TicketState} = {};
```

Comment:
Do you need this variable at all? It seems unused at this point.

Reply:
Like, clearly it's used all over the place, but all those uses are bugs now, aren't they?

Reply:
Are they bugs? It seems a valid way to track state beyond just the roomgames.

Reply:
You're not actually tracking state in this, though.

Reply:
Like you should just be getting the data from the ticket directly.